bambi/backend/terms.py

```diff
@@ -241,6 +241,7 @@ def build(self, nu, invlinks):
             nu = tt.concatenate([np.zeros((data.shape[0], 1)), nu], axis=1)
 
         # Add mean parameter and observed data
+
```

Comment:
was this added on purpose? Otherwise, I would remove it, so the format is the same in all the code.